k8s/README.md

```diff
@@ -252,3 +252,21 @@ This must then be parsed and de-pickled:
 import pickle, base64, json
 pickle.loads(base64.decodestring(json.loads(l[0])['body']))
 ```
+
+## Accessing an EKS cluster
+In order to establish a connection with a cluster running in EKS, you need to first download its KUBECONFIG file. You can do this running:
+```
+aws eks update-kubeconfig --name <cluster-name> --region <region>
```

Comment:
Shaw we add anything about `maws` here?

Reply:
yes, good point. I did not write anything, as theoretically you could log into AWS via other methods. But yeah makes sense to document it